docusaurus/docs/introduction/plugin-types-usage

```diff
@@ -0,0 +1,133 @@
+---
+id: plugin-types-usage
+title: Plugin types and usage
+description: Learn about the different types of Grafana plugins, their usage and configuration.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - usage
+  - provisioning
+  - provision
+  - configuration
+  - configure
+sidebar_position: 1
+---
+
+# Grafana plugin types and usage
+
+:::note
+
+This documentation discusses plugin configuration and usage for plugin developers. General information for plugin users is available at [Plugin management](https://grafana.com/docs/grafana/latest/administration/plugin-management/).
+
+:::
+
+Grafana plugin development allows for many options depending on the type of user experience you want to create. Make a plugin in one of several different types depending on whether you want to offer users a new way of visualizing data, a connection to a new database, or an integrated out-of-the-box experience.  
+
+The following table shows the plugin types currently supported by Grafana on the frontend and backend. You can also see whether or not each type is supported by the [Grafana API](https://grafana.com/api/plugins).
+
+|                |                 |                |                            |
+| -------------- | :-------------: | :------------: | :------------------------: |
+| **Type**       | **Frontend** |   **Backend**  | **SDK-supported** |
+| datasource     |       Yes       | Yes (optional) |             Yes            |
+| app            |       Yes       | Yes (optional) |             Yes            |
+| panel          |       Yes       |       No       |             Yes            |
+| renderer       |        No       |       Yes      |             No             |
+| secretsmanager |        No       |       Yes      |             No             |
```

Comment:
Are we sure that we would like to display the `renderer` and `secretsmanager` types here in case we don't support them and we don't allow external authors to create these type of plugins?

(It feels to me that it's an unnecessary information for them, but can open questions.)

In case we remove those from the list, then we can also remove the "SDK-supported" column I guess.

Reply:
Yes agree with @leventebalogh we shouldn't include these here since not officially supported

Reply:
I've removed the renderer and secretsanager plugin types. The table introduces the plugin types and so makes sense to have it at top (but I've added an "Overview" label to make its purpose clerarer). I've also added a sentence in data-source and app plugins to link to the backend topic.

Reply:
It lists them but I wouldn't say it introduces them :) the information is showing is about whether they include frontend/backend components. Happy to keep here but I would suggest extending the overview to explicitly name the types.

"You can find a plugin type whether you want to offer users a new way of visualizing data with a **panel,** a connection to a new database through a **data source,** or an integrated out-of-the-box experience with an **app**."

Reply:
got it, good suggestion, thanks.